guides/release/components/template-lifecycle-dom-and-modifiers.md

```diff
@@ -321,15 +319,21 @@ Let's start with the HTML we're working with:
 <button type="button">Pause</button>
 ```
 
-It might be tempting to try to handle this by managing the whole set of interactions at the component level. But there are three reasons we might want to tackle this using a modifier instead:
+How should we connect clicking the "Play" and "Pause" to calling the audio tag's `play` and `pause` methods?
+
+While we *could* manage these DOM interactions in the component class (for example, by using `{{did-render}}`), we're better off using a modifier here. It lets us cleanly separate our concerns: the component manages the *state*, and the modifier manages *interactions with the DOM*.
+
+There are three reasons to reach for modifiers for DOM element interactions:
+
+1. A component, by itself, doesn't have direct access to DOM elements. We have to render the page, push an element back up into the component class, and only then can we safely refer to that element. This can sometimes require us to render the component's HTML twice in order for things to start working. Modifiers let us avoid this possible performance issue.
```

Comment:
Thanks for clarifying what you meant by "wire it up". 💯

Reply:
Thanks for catching it as a point of confusion!